AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/UI/ALMAXSwiftUITemplateNativeAdView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  ALMAXSwiftUITemplateNativeAdView.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Matthew Nguyen on 8/1/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import SwiftUI
+import Adjust
+import AppLovinSDK
+
+@available(iOS 13.0, *)
+struct ALMAXSwiftUITemplateNativeAdView: View
+{
+    @ObservedObject private var templateNativeViewModel = ALMAXSwiftUITemplateNativeAdViewModel(adUnitIdentifier: "YOUR_AD_UNIT_ID")
+    
+    var body: some View {
+        VStack {
+            ALMAXNativeTemplateAdViewSwiftUIWrapper(shouldShowAd: $templateNativeViewModel.shouldShowAd,
+                                                    nativeAdLoader: templateNativeViewModel.nativeAdLoader,
+                                                    containerView: templateNativeViewModel.containerView,
+                                                    didLoadNativeAd: templateNativeViewModel.didLoadNativeAd(_:for:),
+                                                    didFailToLoadNativeAd: templateNativeViewModel.didFailToLoadNativeAd(forAdUnitIdentifier:withError:),
+                                                    didClickNativeAd: templateNativeViewModel.didClickNativeAd(_:),
+                                                    didExpireNativeAd: templateNativeViewModel.didExpireNativeAd(_:),
+                                                    didPayRevenue: templateNativeViewModel.didPayRevenue(for:))
+                .frame(width: 300, height: 250)
+            
+            callbacksTable
+                .frame(maxHeight: .infinity)
+            
+            Button {
+                templateNativeViewModel.shouldShowAd = true
+            } label: {
+                Text("Show")
+                    .font(.system(size: 15))
+            }
+        }
+    }
+    
+    var callbacksTable: some View {
+        List(templateNativeViewModel.callbacks) {
+            Text($0.callback)
+        }
+    }
+}
+
+@available(iOS 13.0, *)
+class ALMAXSwiftUITemplateNativeAdViewModel: NSObject, ObservableObject
+{
+    @Published fileprivate var callbacks: [CallbackTableItem] = []
+    @Published var shouldShowAd: Bool = false
+    
+    let adUnitIdentifier: String
```

Comment:
I believe we can remove the stored `adUnitIdentifier` property since we only use it to initialize the `ALMAXSwiftUIAdLoader` (and we can also removed the stored property for `adUnitIdentifier` in `ALMAXSwiftUIAdLoader` too it seems?)